cmd/githubRemoveValidate.go

```diff
@@ -33,6 +33,14 @@ var githubRemoveValidateCmd = &cobra.Command{
 		if err == nil {
 			fmt.Println("gitops not found as expected")
 		}
+		repoMetaphorGo, err := gitWrapper.GetRepo(owner, "metaphor-go")
+		if err == nil {
```

Comment:
is this logic correct? I believe it should be `if err != nil {`

Reply:
yes, this is validate the non-existence of something. If error it is fine.. 

As the next line says: 
```
fmt.Println("gitops not found as expected")
```

Reply:
it is expected to fail on the validation.